Sources/MongoSwift/MongoSwift.swift

```diff
@@ -1,4 +1,40 @@
+import libmongoc
+
 /// Returns a `String` indicating the version of the driver.
 public func version() -> String {
     return "testing"
 }
+
+/// A utility class for libmongoc initialization and cleanup.
+final public class MongoSwift {
+    final class MongocInitializer {
+        static let shared = MongocInitializer()
+
+        private init() {
+            mongoc_init()
+        }
+    }
+
+    /**
+     * Initializes libmongoc.
+     *
+     * This function should be called once at the start of the application
+     * before interacting with the driver.
+     */
+    public static func initialize() {
+        _ = MongocInitializer.shared
```

Comment:
Discarding the result here as we don't actually need a reference to it. Accessing it is enough to trigger lazy initialization of the singleton (i.e. a call to `mongoc_init()`).

Reply:
👍